From 03df2bd59ba7820f49b049fdf49e95cc8fd809d1 Mon Sep 17 00:00:00 2001
From: Alec Leamas <leamas.alec@gmail.com>
Date: Wed, 17 Apr 2024 14:03:26 +0200
Subject: [PATCH 1/3] build: Drop removed
 CoC-1973_2018-InlandECDIS_20220912.pdf

Remove references to pdf file without sources which is
excluded from upstream sources in d/copyright

Not acceptable upstream.

Forwarded: not-needed
---
 CMakeLists.txt | 16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8e69951c1..81896b95a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1752,16 +1752,6 @@ if (APPLE)
     data/opencpn.png PROPERTIES MACOSX_PACKAGE_LOCATION SharedSupport
   )
 
-  set_source_files_properties(
-    data/CoC-909_2013-InlandECDIS_20170308.pdf
-    PROPERTIES MACOSX_PACKAGE_LOCATION SharedSupport
-  )
-
-  set_source_files_properties(
-    data/CoC-1973_2018-InlandECDIS_20220912.pdf
-    PROPERTIES MACOSX_PACKAGE_LOCATION SharedSupport
-  )
-
   set_source_files_properties(
     ${CMAKE_SOURCE_DIR}/data/ocpn-plugins.xml PROPERTIES MACOSX_PACKAGE_LOCATION
                                                          SharedSupport
@@ -1920,8 +1910,6 @@ if (APPLE)
             data/license.html
             data/opencpn.png
             data/ocpn-plugins.xml
-            data/CoC-909_2013-InlandECDIS_20170308s.pdf
-            data/CoC-1973_2018-InlandECDIS_20220912.pdf
             ${SOUND_FILES}
             buildosx/opencpn.icns
             buildosx/MainMenu.xib
@@ -2054,10 +2042,6 @@ if (NOT APPLE)
   file(GLOB LICENSES "${CMAKE_SOURCE_DIR}/COPYING.*")
   install(FILES ${LICENSES} LICENSING DESTINATION "${PREFIX_PKGDATA}")
 
-  install(FILES data/CoC-909_2013-InlandECDIS_20170308s.pdf
-                data/CoC-1973_2018-InlandECDIS_20220912.pdf
-          DESTINATION ${PREFIX_PKGDATA}
-  )
   install(FILES data/copyright DESTINATION ${PREFIX_DATA}/doc/${PACKAGE_NAME})
   install(FILES data/license.html DESTINATION ${PREFIX_PKGDATA})
   install(FILES ${CMAKE_SOURCE_DIR}/data/authors.html
-- 
2.49.0

